### PR TITLE
index.html: close <div> around qrcode correctly

### PR DESCRIPTION
**Pull request by dalgaaf**
Signed-off-by: Danny Al-Gaaf danny.al-gaaf@bisect.de

### CODE REVIEW

**Comment by dalgaaf** [comment]
required to do something like this page: https://dalgaaf.github.io/openstack-summit-barcelona/#/qrcode

related code: https://github.com/dalgaaf/openstack-summit-barcelona/blob/master/index.html#L79

File: index.html [hunk 1]
```diff
@@ -26,9 +26,10 @@
     <div class="reveal">
       <div class="slides">
         <section id="qrcode" data-menu-title="QR code">
-          <div class="qrcode" id="qrcode-talk"/>
```
Comment (fghaas) [inline index.html:29]: Er, that `div` _is_ closed correctly (it's a singleton).

If that does break something for you, I wonder if

``` html
<div class="qrcode" id="qrcode-talk"></div>
```

... would un-break that too?

Comment (dalgaaf) [reply index.html:29]: I checked it. Does somehow help,but not completely.

Check the following screenshots:
1) unchanged original code:
https://drive.google.com/open?id=0BxjHfqaf64A7b1oxYlV4anlBZ28

2) close div with extra tag at the end of the line as you asked for:
https://drive.google.com/open?id=0BxjHfqaf64A7MzQtb3VSTUlNZjQ

3) my pull request: 
https://drive.google.com/open?id=0BxjHfqaf64A7a3o5QnBDZzhvSUE

I assume the reason is the code generating the qrcode. The href needs to be part of the div block to work as expected.

Comment (fghaas) [reply index.html:29]: Yeah but the correct rendering **is** actually (2). QR code at the top, then the link, then whatever comes after the link. If you want it the other way around (link first), then the correct HTML would be

``` html
<section id="qrcode" data-menu-title="QR code">
  <p><a href="https://example.com/link/to/this/talk"
        target="_blank"
        id="talk">https://example.com/link/to/this/talk</a></p>
  <div class="qrcode" id="qrcode-talk"></div>
</section>
```

So I do totally concede that the code as it is is broken and doesn't actually do what I originally intended it to do (but then forgot about), which is to render the link **below** the QR code, as a caption. But the correct fix is to properly close the `<div>` where it is, not to wrap it around the link.

**Pull request closed without merging**